src/securityPolicy.c

```diff
@@ -547,6 +547,35 @@ security_policy_t policyForSignTxCertificateStakePoolRetirement(
 	DENY(); // should not be reached
 }
 
+security_policy_t policyForSignTxStakePoolRegistrationInit(
+        sign_tx_signingmode_t txSigningMode,
+        size_t numOwners
+)
+{
+	switch (txSigningMode) {
+	case SIGN_TX_SIGNINGMODE_POOL_REGISTRATION_OWNER:
+		// there should be exactly one owner given by path for which we provide a witness
+		DENY_IF(numOwners == 0);
+		ALLOW();
+		break;
+
+	case SIGN_TX_SIGNINGMODE_POOL_REGISTRATION_OPERATOR:
+		ALLOW();
+		break;
+
+	case SIGN_TX_SIGNINGMODE_ORDINARY_TX:
+	case SIGN_TX_SIGNINGMODE_MULTISIG_TX:
+		DENY();
+		break;
+
+	default:
+		ASSERT(false);
+		break;
+	}
+
+	DENY();
```

Comment:
```suggestion
	DENY(); // should not be reached
```

Reply:
Done, but can't merge until it's resolved.